src/Repository/Game/GameRepository.php

```diff
@@ -123,4 +87,32 @@ public function findLatestActiveByChat(Chat $chat): ?Game
 
         return $qb->getQuery()->getOneOrNullResult();
     }
+
+    /**
+     * @param Chat $chat
+     *
+     * @return Game
+     */
+    public function findActiveOrCreatedTodayByChat(Chat $chat): ?Game
+    {
+        $from = (new DateTime())->setTime(0, 0, 0);
+        $to = (new DateTime())->setTime(23, 59, 59);
```

Comment:
in the future it is necessary to use https://github.com/beberlei/DoctrineExtensions/blob/master/src/Query/Mysql/Date.php